variables.tf

```
@@ -76,6 +76,12 @@ variable "realm" {
   default     = "EXAMPLE.COM"
 }
 
+variable "root_disk_size" {
+  type        = number
+  description = "The size of the IPA instance's root disk in GB."
```

Just mirroring the previous suggestion.

```suggestion
  description = "The size of the IPA instance's root disk in GiB."
```